decidim-core/app/models/decidim/searchable_rsrc.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "pg_search"
+
+module Decidim
+  # A searchable Resource
+  class SearchableRsrc < ApplicationRecord
+    include PgSearch
+
+    belongs_to :organization,
+               foreign_key: "decidim_organization_id",
+               class_name: "Decidim::Organization"
+    belongs_to :scope,
+               foreign_key: "decidim_scope_id",
+               class_name: "Decidim::Scope",
+               optional: true
+    belongs_to :resource, polymorphic: true
+    belongs_to :decidim_participatory_space, polymorphic: true
+
+    validates :locale, uniqueness: { scope: :resource }
+
+    pg_search_scope :global_search,
+                    against: { content_a: "A", content_b: "B", content_c: "C", content_d: "D" },
```

Comment:
Can you add a comment (maybe improve the class docs) explaining what `content_a/b/c/d` mean? Seems rather cryptical to me 😕